tests/PHPStan/Analyser/data/bug-8272.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+use function PHPStan\Testing\assertType;
+
+assertType('int<1, 5>', mt_rand(1, 5));
```

Comment:
This test file is not included from any test-class?

Should be yielded from NodeScopeResolverTest

Reply:
And maybe tested with no arguments.